Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -399,15 +399,16 @@ extension CarPlayManager: CPApplicationDelegate {
         guard let carPlayMapViewController = carPlayMapViewController else {
             return nil
         }
-        var mapButtons = [
+        
+        let panMapButton = carPlayMapViewController.panningInterfaceDisplayButton(for: mapTemplate)
+        carPlayMapViewController.panMapButton = panMapButton
+        
+        let mapButtons = [
             carPlayMapViewController.recenterButton,
+            panMapButton,
             carPlayMapViewController.zoomInButton,
             carPlayMapViewController.zoomOutButton
         ]
-        let panMapButton = carPlayMapViewController.panMapButton ??
-            carPlayMapViewController.panningInterfaceDisplayButton(for: mapTemplate)
```

Comment:
Not sure why before we were using this check, doesn't seem to significantly help with performance, and is the root case of this issue.